CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+In order to contribute,
+ - Fork this repo
+ - Create a Branch on your forked repo
+ - Developer features and Create a Git Commit
+ - Push your changes
+ - Create a Pull Request
+
+# Formatting
+Before creating a Pull Request, make sure to scripts are formatted and aligned. The formatting rules are controlled by python packages,
```

Comment:
Please fix '...make sure to scripts are formatted' to 'make sure that the scripts are formatted'